runtime/doc/options.txt

```diff
@@ -3530,6 +3530,10 @@ A jump table for the options with a short description can be found at |Q_op|.
 			jumplist and then jumping to a location.
 			|jumplist-stack|
 
+	  view          When moving through the jumplist, changelist, and
+			|alternate-file| try to restore the |mark-view| in
+			which the jump occurred.
```

Comment:
Why not also make it apply to `'x` ? Then we can drop the `z'` / `[count]'` variants, at least until we get strong feedback showing a use-case for _not_ restoring the view.